docs/landing-zone-v2/README.md

```diff
@@ -364,7 +364,7 @@ cd pbmm-landingzone
 1. Customize Packages
 
     Review and customize all packages' `setters.yaml` with the unique configuration of your landing zone.  
-    For example "core-landing-zone" will have the same [setters.yaml](https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/main/solutions/landing-zone-v2/setters.yaml) as in the repo in the root of the pkg directory.
+    For example "core-landing-zone" will have the same [setters.yaml](https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/main/solutions/core-landing-zone/setters.yaml#L41) as in the repo in the root of the pkg directory.
```

Comment:
might be worth removing reference to line 41 (`#L41`) to avoid updating this link if that file's content changes in the future?

Reply:
was going to skip to the start of the content - but yes lets start at the root